### PR TITLE
Added Key Icons

### LINKED ISSUE #113: Key Icons can be used in README
#### This issue is about Key Icons in README.

- [x] I searched or browsed the repo’s other issues to ensure this is not a duplicate.
- [x] This issue is for enhancement/bug fixing.
- [ ] This issue is an addition to JSON database.
- [ ] I have gone through [Contribution Guidelines](https://github.com/Anshuman-Verma/ongaku/blob/master/CONTRIBUTING.md).
- [x] Assign this issue to me.

### PR DESCRIPTION
**Pull request by aashutoshrathi**
Fixes #113 

#### This PR is for Key Icons.

- [x] This issue is for enhancement/bug fixing.
- [ ] This issue is an addition to JSON database.
	- [ ] Pull Request Title is of form `Added [Anime Name]`.
- [x] This addition is already on ongaku.
	- [x] This PR is an improvement to current addition.
- [x] I have checked my addition locally and it is working and won't break existing entries.
- [x] I have gone through [Contribution Guidelines](https://github.com/Anshuman-Verma/ongaku/blob/master/CONTRIBUTING.md).
- [x] This pull request will be closed if I failed to update it even once in a week.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,17 +17,17 @@
...
-| SPACE | Pause or play |
-| N | Play next track |
-| R | Replay current track |
-| F | Toggle fullscreen |
-| L | Play last played track / Track on the end of the list |
-| UP | Increments volume by 10% |
-| DOWN | Decrements volume by 10% |
-| LEFT | Jump 10 seconds backward |
-| RIGHT | Jump 10 seconds forward |
+| <img src="https://www.techonthenet.com/clipart/keyboard/images/space_key_m.png" width="35"> Spacebar| Pause or play |
```
Comment (anshumanv) [inline README.md]: Take that spacebar word out, please

**Review by anshumanv — COMMENTED** [review]
Looks amazing 😱

**Comment by Monal5031** [comment]
@aashutoshrathi @Anshuman-Verma Its better to use fa icons here too, as the links may get archived or become dead in future but the fa icons won't. IT will be much better to use it that way.
fa to be used : `fa-arrow-up`

**Comment by aashutoshrathi** [comment]
@Anshuman-Verma I need help with space bar icon, this one is not best, and also is size good?

**Comment by anshumanv** [comment]
Yea @Monal5031 , actually fa icons should hold well here, probably switch to fa icons @aashutoshrathi ?

**Comment by aashutoshrathi** [comment]
@Monal5031 I checked FA at the very beginning but they don't appeal on first look that we are talking about keys.

**Comment by Monal5031** [comment]
http://fontawesome.io/cheatsheet/
Check out all the fa icons here @aashutoshrathi @Anshuman-Verma

**Comment by anshumanv** [comment]
fa icons need an import right ? I'm not sure if gh md supports it, have u tried it personally @Monal5031 ?

**Comment by anshumanv** [comment]
Have any other alternatives in mind for spacebar @aashutoshrathi ? (If we are not going for fa)

**Comment by Monal5031** [comment]
@Anshuman-Verma md supports markups too.
It was working the last time I did.

**Comment by aashutoshrathi** [comment]
> @Anshuman-Verma md supports markdown also html is markdown.

WTF ? HTML is markdown -__- ? It's markup right.
In case you are talking about pasting Unicode icons, its good. 👍

**Comment by anshumanv** [comment]
So u mean font-awesome library is already imported in gh md ? @Monal5031

**Comment by Monal5031** [comment]
@aashutoshrathi My bad, but it does support it.
@Anshuman-Verma We can give it a try but last I tried it really worked.

**Comment by aashutoshrathi** [comment]
Ok, @Monal5031 let me try.

**Review by anshumanv — APPROVED** [review]
👍

**Pull request merged**